 int32 goal_id
 int32 x_image_frame
 int32 y_image_frame
 int32 xmin
int32 xmax
int32 ymin
int32 ymax
 float32 distance